eng/source-build/build.sh

```diff
@@ -83,4 +83,5 @@ fi
 
 ReadGlobalVersion Microsoft.DotNet.Arcade.Sdk
 export ARCADE_VERSION=$_ReadGlobalVersion
+export NUGET_PACKAGES=$scriptroot/../../artifacts/source-build/self/package-cache/
```

Comment:
Is this used/exist in the full repo source-build?

Reply:
It is set in full source-build: https://github.com/dotnet/dotnet/blob/8528c489bcbaeff232b906a3c784016f2de74fe2/build.sh#L248

I think I need to condition this for repo build only.

Reply:
Resolved in https://github.com/NuGet/NuGet.Client/pull/5190/commits/eaaaebcd571a201b0bda7ae3e49767824975471a